tests/vmi_configuration_test.go

```diff
@@ -165,7 +165,16 @@ var _ = Describe("Configurations", func() {
 						kubev1.ResourceMemory: resource.MustParse("64M"),
 					},
 				}
-				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				// Retrying 5 times with its interval set to one second, then if you still succeeds
```

Comment:
maybe just add a sleep here for 5 seconds before doing the create instead?

Reply:
Since it should update fast I suggested a retry, since it should always succeed on the first or second try, but sleep would work too.

Reply:
I'll push a fix with no sleep, thanks.